tableau-mkt/tech-assess-2021q3
# Coded tableau custom module for take home assessment

**To test:**

1. run `drush en tableau` or install via /admin/modules
2. navigate to any node
3. within sidebar_first region, click button titled 'Click me!' & make sure table with accessibility violations renders below button element
